pkg/command/server.go

```diff
@@ -35,15 +34,13 @@ func Server(cfg *config.Config) cli.Command {
 				cfg.HTTP.Root = strings.TrimSuffix(cfg.HTTP.Root, "/")
 			}
 
+			cfg.Phoenix.Config.Apps = c.StringSlice("web-config-apps")
```

Comment:
I personally put this assignment normally into `Before`, than at least it feels more consistent until StringSliceFlag also gets the Destination keyword.

Reply:
it is in the before callback

Reply:
Oh.... GitHub tricked me, sorry :D

Reply:
hahaha